applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

```diff
@@ -330,7 +330,7 @@ After programming the application and all the prerequisites to your development
 1. |connect_kit|
 #. Connect to the kit with a terminal emulator (for example, LTE Link Monitor). See :ref:`lte_connect` for more information.
 #. Reset the development kit.
-#. Observe in the terminal window that the development kit starts up in the Secure Partition Manager and that the application starts.
+#. Observe in the terminal window that the development kit starts up in the Trusted Firmware-M and that the application starts.
```

Comment:
I wonder if this would read better if `Trusted Firmware-M` were followed by something like `image`, `environment`, `module`, or `subsystem`?

Reply:
Of those, environment works best in my opinion.